edx_rest_framework_extensions/permissions.py

```diff
@@ -144,6 +144,26 @@ def _get_user_filter(self, request):
         return None
 
 
+class LoginRequired(IsAuthenticated):
```

Comment:
Nit: It's kind of odd that this `Permission` class is very far away from its sister `Authentication` class.

Reply:
100% Agree.  However, I thought the best short-term was to put it with all the other JWT based permission classes.

Note: We did a big refactor of edx-drf-extensions, but the refactor got too large, so we decided to put off refactoring the permissions.  I think we should finish that work at some point, but not as part of this PR.